opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -345,37 +345,54 @@ def _import_id_generator(id_generator_name: str) -> IdGenerator:
     raise RuntimeError(f"{id_generator_name} is not an IdGenerator")
 
 
-def _initialize_components(auto_instrumentation_version):
-    trace_exporters, metric_exporters, log_exporters = _import_exporters(
-        _get_exporter_names("traces"),
-        _get_exporter_names("metrics"),
-        _get_exporter_names("logs"),
+def _initialize_components(**kwargs):
```

Comment:
since you are going to use it from outside I think it would be better to make parameters explicit with types and everytihng

Reply:
On second thought, I'm seeing that _BaseConfigurator and _OTelSDKConfigurator that custom configurators are likely to extend already have kwargs in their method signature. It's just that only one args, auto_instrumentation_version, is currently being used. I think in order to minimize api and signature changes, it would be best to leave as kwargs and add documentation and/or docstrings. But open to further input.

Reply:
Looks good